services/operator-date-interval-parser.js

```diff
@@ -95,7 +95,7 @@ function OperatorDateIntervalParser(value, timezone) {
       return {
         $gte: moment().subtract(match[1] - 1, 'days').startOf('day')
                 .add(offsetHours, 'h').toDate(),
-        $lte: moment().add(offsetHours, 'h').toDate()
+        $lte: moment().toDate()
```

Comment:
Can you explain why the end of the period does not take the timezone into account?